src/content-handlers/iiif/modules/uv-pagingheaderpanel-module/PagingHeaderPanel.ts

```diff
@@ -407,13 +407,13 @@ export class PagingHeaderPanel extends HeaderPanel<
       $(this).select();
     });
 
-    this.$searchButton.onPressed(() => {
+    this.onAccessibleClick(this.$searchButton, () => {
       if (this.options.autoCompleteBoxEnabled) {
         this.search(this.$autoCompleteBox.val());
       } else {
         this.search(this.$searchText.val());
       }
-    });
+    }, true, true);
```

Comment:
Another anchor that has changed to a button...

Reply:
This should be resolved now